src/osd/ECBackend.cc

```diff
@@ -229,6 +229,18 @@ void ECBackend::_failed_push(const hobject_t &hoid,
   get_parent()->finish_degraded_object(hoid);
 }
 
+// Only called when CONFIG config option set
+void ECBackend::ignore_failed_push(const hobject_t &hoid)
+{
```

Comment:
should we call get_parent()->recover_got() to clear the primary's missing state too?

Reply:
I think that the new missing_drop_replicas() is analogous to recover_got().  The difference that instead of using got() which clears missing for a specific version, it uses rm() using eversion_t::max() which clears all missing versions.